src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx

```diff
@@ -119,12 +128,11 @@ const BottomSheetBackdropComponent = ({
       <Animated.View
         style={containerStyle}
         pointerEvents={pointerEvents}
-        accessible={true}
-        accessibilityRole="button"
-        accessibilityLabel="Bottom Sheet backdrop"
-        accessibilityHint={`Tap to ${
-          typeof pressBehavior === 'string' ? pressBehavior : 'move'
-        } the Bottom Sheet`}
+        accessible={_providedAccessible ?? undefined}
+        accessibilityRole={_providedAccessibilityRole ?? undefined}
+        accessibilityLabel={_providedAccessibilityLabel ?? undefined}
+        accessibilityHint={_providedAccessiblityHint ?? undefined}
+        {...rest}
       >
         {children}
       </Animated.View>
```

Comment:
I think you also need to add the accessibility props to the `<Animated.View>` below:
```
    <Animated.View
      style={containerStyle}
      pointerEvents={pointerEvents}
      accessible={_providedAccessible ?? undefined}
      accessibilityRole={_providedAccessibilityRole ?? undefined}
      accessibilityLabel={_providedAccessibilityLabel ?? undefined}
      accessibilityHint={_providedAccessibilityHint ?? undefined}
      {...rest}
    >
      {children}
    </Animated.View>
```
Maybe add this to a local const and use that because it is the exact same component as is used inside `<TapGestureHandler/>` when `pressBehavior !== 'none'`.